bin/pygrb/pycbc_pygrb_page_tables

```diff
@@ -44,173 +46,139 @@ __version__ = pycbc.version.git_verbose_msg
 __date__ = pycbc.version.date
 __program__ = "pycbc_pygrb_page_tables"
 
-
 # =============================================================================
 # Functions
 # =============================================================================
-# Function to load trigger data
-def load_data(input_file, vetoes, ifos, opts, injections=False):
-    """Load data from a trigger/injection file"""
-
-    # Initialize the dictionary
-    data = {}
-    data['trig_time'] = None
-    data['coherent'] = None
-    data['reweighted_snr'] = None
-
-    if input_file:
-        if injections:
-            logging.info("Loading injections...")
-            # This will eventually become ppu.load_injections()
-            trigs_or_injs = ppu.load_triggers(input_file, vetoes)
-        else:
-            logging.info("Loading triggers...")
-            trigs_or_injs = ppu.load_triggers(input_file, vetoes)
-            
-        data['trig_time'] = trigs_or_injs['network/end_time_gc'][:]
-        num_trigs_or_injs = len(data['trig_time'])
-        data['coherent'] = trigs_or_injs['network/coherent_snr'][:]
-        data['reweighted_snr'] = trigs_or_injs['network/reweighted_snr'][:]
-
-        if injections:
-            logging.info("%d injections found.", num_trigs_or_injs)
-        else:
-            logging.info("%d triggers found.", num_trigs_or_injs)
-
+def additional_injection_data(data, ifos):
+    """Provides data with chirp masses and effective distances"""
+    data['mchirp'] = mchirp_from_mass1_mass2(data['mass1'],
+                                             data['mass2'])
+    eff_dist = 0
+    for ifo in ifos:
+        antenna = Detector(ifo)
+        data['eff_dist_%s' % ifo] = antenna.effective_distance(
+                                    data['distance'],
+                                    data['ra'],
+                                    data['dec'],
+                                    data['polarization'],
+                                    data['tc'],
+                                    data['inclination']
+                                    )
+        eff_dist += 1.0 / data['eff_dist_%s' % ifo]
+    data['eff_dist'] = 1.0 / eff_dist
     return data
 
 
-def get_column(table, column_name):
-    """Wrapper for glue.ligolw.lsctables.MultiInspiralTable.get_column
-    method. Easier for h5py switch. Note: Must still replace
-    .get_sigmasqs() and .get_snglsnr() methods throughout the scripts.
+def load_missed_found_injections(hdf_file, ifos, snr_threshold, bank_file,
+                                 trig_data, background_bestnrs=None):
```

Comment:
```suggestion
                                 trig_data, background_bestnrs=None):
```

Reply:
(we do not need `trig_data` when dealing with injections; see fixes below in this function).